C7/LogManager.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Serilog;
+using Serilog.Templates;
+using Serilog.Templates.Themes;
+
+public class LogManager
+{
+    static LogManager()
```

Comment:
for any runtime initialisation that must be done before some feature is used, AFAIK we don't have an "entry point" for C7, so I think using a static factory class to ensure the initialisation happens first is the best way to solve this